### PR TITLE
Fix #1747: Project title appears on editor header for anonymous projects

### LINKED ISSUE #1747: Missing "Project Title" on editor header
screen shot 2017-02-16 at 3 08 49 pm

On our [staging server](https://bramble.mofostaging.net) I'm noticing that we suddenly don't have any text for the project title.  Above, the top is staging, bottom is production.

### PR DESCRIPTION
**Pull request by omytryniuk**
I have fixed the issue. Now the title appears for anonymous projects. 
fix1

Let me know, please, if you have any questions or you would like to change something. 
Thank you very much!

### CODE REVIEW

File: public/editor/scripts/editor/js/fc/project-rename.js [hunk 1]
```diff
@@ -133,7 +133,10 @@ define(function(require) {
         editingComplete(context);
         analytics.event("ProjectRenamed");
 
-        context.publisher.showUnpublishedChangesPrompt();
+        //Check whether the project is not anonymous		
+        if(context.publisher)
```
Comment (humphd) [inline public/editor/scripts/editor/js/fc/project-rename.js]: Always wrap your `if` in `{...}`, even for 1 line.

File: public/editor/scripts/editor/js/fc/project-rename.js [hunk 2]
```diff
@@ -133,7 +133,10 @@ define(function(require) {
...
+          context.publisher.showUnpublishedChangesPrompt();		
+
```
Comment (humphd) [inline public/editor/scripts/editor/js/fc/project-rename.js]: Remove this blank line.

File: public/editor/scripts/editor/js/fc/project-rename.js [hunk 3]
```diff
@@ -133,7 +133,10 @@ define(function(require) {
         editingComplete(context);
         analytics.event("ProjectRenamed");
 
-        context.publisher.showUnpublishedChangesPrompt();
+        //Check whether the project is not anonymous
```
Comment (humphd) [inline public/editor/scripts/editor/js/fc/project-rename.js]: Space after `//`

**Comment by omytryniuk** [comment]
@humphd Thank you very much! 
Travis did not pass first time because I did not wrap the if statement in {}. Interesting :)

**Comment by flukeout** [comment]
Checking it out now.

**Comment by flukeout** [comment]
Great, works as expected! Thanks very much @omytryniuk

**Pull request merged**